.github/ISSUE_TEMPLATE/release_tracker.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Release Tracker Issue"
+description: "Track component releases and update manifests before the deadline"
+title: "[Release Tracker] - <Insert Release Name>"
+body:
+  - type: input
+    id: release_date
+    attributes:
+      label: "📅 Planned Release Date"
+      description: "Enter the planned release date (e.g., Mar 21st, 2025)"
```

Comment:
not sure i understand this.
we set code freeze day for component teams to submit their changes with certain date, normally on a friday
then the next Monday is the planned release date.
what is Manifests update deadline and planned release date mean here?

e.g odh 2.26.0 has code freeze on 14th March 2025, and release date on 17th March 2025.
how this example 21th March 2025 come in here?

Reply:
you are right
Manifests update deadline = code freeze
 planned release date = the Monday we do the release
 And my mistake it should be 17th. changing now

Reply:
anyway i think we need to be more
code freeze is code freeze,  stick with the general term, not try to invent new name "manifests udpate deadline"

Reply:
ok let me update

Reply:
done now 👍